README.md

```diff
@@ -2,25 +2,21 @@
 
 __Copyright (c) 2007 - 2021 Unity Project by Mike Karlesky, Mark VanderVoord, and Greg Williams__
 
-Welcome to the Unity Test Project, one of the main projects of ThrowTheSwitch.org.
-Unity Test is a unit testing framework built for C, with a focus on working with embedded toolchains.
+Welcome to Unity Test Project, one of the main projects of ThrowTheSwitch.org.
+Unity Test is a unit testing framework built for C, which focuses on working with embedded toolchains.
 
-This project is made to test code targetting microcontrollers big and small.
-The core project is a single C file and a pair of headers, allowing it to the added to your existing build setup without too much headache.
-You may use any compiler you wish, and may use most existing build systems including Make, CMake, etc.
+This project is made to test code targeting microcontrollers big and small.
+This project is a single C file and a pair of headers, allowing you to use this with his existing build setup without too much headache.
+You can use any compiler, and may use most of the existing build systems including Make, CMake, etc.
```

Comment:
"targetting" is a spelling error - good change.

Changing "The core" to "This" is changing the semantics, not sure why this change is needed.

Replacing "allowing it to the added to your existing" with "allowing you to use this with his existing" is bad, maybe it should be "allowing you to use this with your existing" but I'm not sure it is better than the original text.